plugins/unifying/unifying.quirk

```diff
@@ -36,3 +36,7 @@ Flags = is-bootloader,is-texas
 FirmwareSizeMin = 0x4000
 CounterpartGuid = HIDRAW\VEN_046D&DEV_C52B
 InstallDuration = 18
+
+# Possible HID++ v2.0 peripheral device
+[DeviceInstanceId=HIDRAW\VEN_046D]
+Plugin = unifying
```

Comment:
We need to make sure this doesn't grab the hidraw node from the receiver. Can you confirm the `[DeviceInstanceId=HIDRAW\VEN_046D&DEV_C52B]` entry will be selected instead of this one?

Reply:
The instance I'd only marks the device as "interesting" -- it's up to the plugin to test the device for suitability before writing to it.

Reply:
Then we need to make sure we choose the correct one in the plugin.